app/services/organizations.js

```diff
@@ -8,7 +8,7 @@ export default Service.extend({
     this.organizations = Object.create(organizations);
   },
 
-  list: computed('organizations', function () {
+  list: computed('organizations', function listOrganization() {
```

Comment:
this is a file you added in another commit.  the lint fixup should be in the original commit where you add the file.

Run the linter on your 'real' commit and fix errors there.

The lint fixup commit should then only include additional style changes to lines of code which you did not modify in your 'real' commit.